app/src/main/java/com/example/packpals/models/Itinerary_Item.kt

```diff
@@ -1,7 +1,17 @@
 package com.example.packpals.models
 
+import com.google.firebase.firestore.GeoPoint
+import java.util.Date
+
 data class Itinerary_Item(
-    val location: String,
-    val date: String,
-    val forecast: String
-)
\ No newline at end of file
+    val tripId: String? = null,
+    val date: Date? = null,
+    var forecast: String? = null,
```

Comment:
I think forecast should probably not be in the data class, maybe just call the weather api for the itinerary items for up to date forecast whenever itinerary list is opened and not store this at all